### DISCUSSION TITLE
Bulk De-Archive not working

### DISCUSSION THREAD

**Discussion by naveeddil** [xlg8/APMIS-Project#665] [General]
Nothing Happens when you click yes button

@RaziaMahmodi @bnygren

![APMIS Bulk De Archieve](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/e04305e7-21a0-4336-9329-def1ac5a45f2)

**Comment by RaziaMahmodi** [comment]
@naveeddil Dev team sis working on it.
